digits/model/images/classification/views.py

```diff
@@ -190,11 +190,8 @@ def create():
                         elif epoch == -1:
                             pretrained_model = old_job.train_task().pretrained_model
                         else:
-                            for filename, e in old_job.train_task().snapshots:
-                                if e == epoch:
-                                    pretrained_model = filename
-                                    break
-
+                            # verify snapshot exists
+                            pretrained_model = old_job.train_task().get_snapshot(epoch, download=True)
```

Comment:
According to git history, @gheinrich was the one that added the line. @gheinrich can you comment on this please?